katsdptelstate/backend.py

```diff
@@ -160,12 +160,20 @@ def monitor_keys(self, keys):
 
         Returns a generator. The first yield from the generator is a no-op.
         After that, the caller sends a timeout and gets back an update event.
-        Each update event is a tuple, of the form (key, value) or (key, value,
-        timestamp) depending on whether the update is to an immutable or a
-        mutable key; or of the form (key,) to indicate that a key may have
-        been updated but there is insufficient information to provide the
-        latest value. If there is no event within the timeout, returns
-        ``None``.
+        Each update event is a tuple of the form (type, key, value, ...). The
```

Comment:
Given the fairly substantial amount of positional indexing going on interpreting the tuple, it seems we might have reached the point that have a KeyUpdate object might be more efficient in the long term.

Reply:
Simon, are you shunning your ffuilib ways? 😜

Reply:
Done.